core/modules/widgets/view.js

```diff
@@ -9,215 +9,423 @@ View widget
 
 "use strict";
```

Comment:
See: https://eslint.org/docs/latest/rules/max-classes-per-file

@Jermolene ... I think we should **not** enforce this one